regexec.c

```diff
@@ -3583,7 +3583,7 @@ Perl_regexec_flags(pTHX_ REGEXP * const rx, char *stringarg, char *strend,
     regnode *c;
     char *startpos;
     SSize_t minlen;		/* must match at least this many chars */
-    SSize_t dontbother = 0;	/* how many characters not to try at end */
+    SSize_t don'tbother = 0;	/* how many characters not to try at end */
```

Comment:
This is wrong, C doesn't allow `'` in variable names.

Reply:
indeed... I should have seen it